scripts/on-prem-archive.sh

```diff
@@ -21,6 +21,10 @@
 # HAB_ON_PREM_BOOTSTRAP_KEEP_ARCHIVE_FILE: Sometimes you don't want to delete the
 # archive file you just spent many minutes creating. Setting this keeps the file.
 #
+# HAB_ON_PREM_BOOTSTRAP_NO_AWS: This controls whether the script will upload the
```

Comment:
Looks good.. small feedback that it might be better to make this a "no upload" flag instead of AWS specific - eg, "HAB_ON_PREM_BOOTSTRAP_NO_UPLOAD"

Reply:
sure.  sounds reasonable.